content/en/agent/faq/kubernetes-state-cluster-check.md

```diff
@@ -0,0 +1,113 @@
+---
+title: How do I run the kubernetes_state check as a cluster check?
+kind: faq
+further_reading:
+- link: "agent/kubernetes"
+  tag: "Documentation"
+  text: "Kubernetes with Datadog"
+- link: "agent/autodiscovery"
+  tag: "Documentation"
+  text: "Using Autodiscovery with Kubernetes and Docker"
+- link: "agent/autodiscovery/clusterchecks"
+  tag: "Documentation"
+  text: "Running Cluster Checks with Autodiscovery"
+---
+
+The `kubernetes_state` check, which monitors `kube-state-metrics`, is enabled automatically by [Autodiscovery][1]. In larger Kubernetes clusters, this check may consume too much RAM. In this case, consider running the `kubernetes_state` check as a [cluster check][2].
+
+## Configuration
+
+1. Deploy `kube-state-metrics` with cluster check annotations on the service:
```

Comment:
Please change to

```
1. Deploy `kube-state-metrics` with cluster check annotations on the Kubernetes service:


```
apiVersion: v1
kind: Service
metadata:
  annotations:
    # ... others
    ad.datadoghq.com/service.check_names: '["kubernetes_state"]'
    ad.datadoghq.com/service.init_configs: '[{}]'
    ad.datadoghq.com/service.instances: |
      [
      {
          "kube_state_url": "http://%%host%%:8080/metrics",
          "prometheus_timeout": 30,
          "min_collection_interval": 30,
          "send_pod_phase_service_checks": false,
          "telemetry": true
      }
      ]
```